comment/models/comments.py

```diff
@@ -13,6 +14,11 @@ class Comment(models.Model):
     object_id = models.PositiveIntegerField()
     content_object = GenericForeignKey('content_type', 'object_id')
     content = models.TextField()
+    urlhash = models.CharField(
+        max_length=len(settings.COMMENT_URL_PREFIX + settings.COMMENT_URL_SUFFIX) + settings.COMMENT_URL_ID_LENGTH,
```

Comment:
We agreed to use `max_length=50`.
I just realized that there will be significant issue with this design, whenever the developer change the `COMMENT_URL_*` settings, the `max_lenght` will be changed and the model will be updated, new migration file will be ceated and this will mess the migration history up.

Reply:
The migration history won't be messed up as long as these changes are made after running our migrations. I have used static values in our migrations.

Reply:
It does really. I just changed the settings `COMMENT_URL_ID_LENGTH = 20` and run migrations and new migrations file was created here:
```python
# Generated by Django 3.0.7 on 2020-07-04 11:37

from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('comment', '0008_comment_urlhash'),
    ]

    operations = [
        migrations.AlterField(
            model_name='comment',
            name='urlhash',
            field=models.CharField(editable=False, max_length=28, unique=True),
        ),
    ]

```

Reply:
This doesn't mess up with the migration history. The file is generated since the constraints in the database have to be changed. The history is intact.

Reply:
I am not sure why we are arguing on this?! I don't want a migration file to be created whenever the developer change his settings, this is absolutely not a good design.

Reply:
BTW, the developers who use our app should not be able to create or change any of the migration files.

Reply:
> I am not sure why we are arguing on this?! I don't want a migration file to be created whenever the developer change his settings, this is absolutely not a good design.

Okay. 50 is a reasonable length here.

Also, It would be better if we use gender-neutral pronouns like they instead of his/her.

Reply:
Maybe we also need to put a test somewhere that checks this constraint and asks the developer to adhere if set values that go beyond 50. I was not sure where to put this in a file that is triggered when the settings are changed.

Reply:
> BTW, the developers who use our app should not be able to create or change any of the migration files.

As long as the code is open-source they obviously can :-p Although, I get your point in this context.